pymess/models/common.py

```diff
@@ -1,7 +1,19 @@
-from django.db.models import Manager
+from chamber.models import SmartModel
+from chamber.utils.datastructures import ChoicesNumEnum
+from django.contrib.contenttypes.fields import GenericForeignKey
 from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.db.models import Manager
+from django.template import Template, Context
+from django.template.exceptions import TemplateSyntaxError, TemplateDoesNotExist
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Proč `ugettext_lazy` jako `_` a ne `_l`?

Reply:
protože _l by default nefunguje, je to naše rozšíření